frontend/scss/vendor/mathigon.scss

```diff
@@ -403,3 +386,29 @@ x-bio, x-gloss {
 .math {
   font-size: 1rem;
 }
+
+
+button.a, label a, ol a, p a, x-step p a, ul a {
+  color: $link-color-tertiary;
+  background: initial;
+
+    &.c-sidebar__entry {
+      color: unset;
+    }
+
+    &::before {
+      background: initial;
+    }
+
+    &:hover, &:focus {
+      color: $link-color-tertiary;
+      text-decoration: underline;
+      background: initial;
+      box-shadow: none;
+    }
+
+    &::after {
+      background: initial;
+      background-image: url("data:image/svg+xml;utf8,%3Csvg%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%20width%3D%228%22%20height%3D%2214%22%3E%3Cpath%20fill%3D%22none%22%20stroke%3D%22white%22%20stroke-width%3D%222px%22%20d%3D%22M2%202L6%207L2%2012%22/%3E%3C/svg%3E");
```

Comment:
we should remove the `background-image`. it adds the `>` character which is not needed:

you can get a glimpse of it when you hover over a link with surrounding text:

<img width="156" alt="image" src="https://user-images.githubusercontent.com/13156555/134935065-619ecfd9-343b-461b-a181-1e46cb33267a.png">

Reply:
ah I did wonder what that did